pkg/proxy/proxy.go

```diff
@@ -161,12 +161,45 @@ func filterList(rels []*release.Release) []*release.Release {
 	return uniq
 }
 
+func getStatuses(statusQuery string) []release.Status_Code {
+	if statusQuery == "" {
+		// Default case
+		return []release.Status_Code{
+			release.Status_DEPLOYED,
+			release.Status_FAILED,
+		}
+	} else if strings.Contains(statusQuery, "all") {
+		return releaseStatuses
```

Comment:
let's call these allReleaseStatuses and add a comment that it matches Helm list https://github.com/helm/helm/blob/master/cmd/helm/list.go#L214